imagetest/test_suites/image_boot/setup.go

```diff
@@ -0,0 +1,15 @@
+package imageboot
+
+import "github.com/GoogleCloudPlatform/guest-test-infra/imagetest"
+
+// Name is the name of the test package. It must match the directory name.
+var Name = "image_boot"
+
+// TestSetup sets up the test workflow.
+func TestSetup(t *imagetest.TestWorkflow) error {
+	vm, err := t.CreateTestVM("image-boot-test")
```

Comment:
please name the vm 'vm' unless you're going to use the value. the workflow will already add image-boot to it.